gym_solo/envs/solo8v2vanilla.py

```diff
@@ -117,7 +107,18 @@ def reset(self, init_call: bool = False) -> solo_types.obs:
     Returns:
       solo_types.obs: The initial observation of the space.
     """
-    self.client.removeBody(self.robot)
+    self.client.resetSimulation()
+    self.client.setGravity(*self.config.gravity)
+
+    if self.config.dt:
+      self.client.setPhysicsEngineParameter(fixedTimeStep=self.config.dt, 
+                                            numSubSteps=1)
+    else:
+      self.client.setRealTimeSimulation(1)
+
+    self.client_configuration()
+
+    self.plane = self.client.loadURDF('plane.urdf')
```

Comment:
Pybullet has a memory leak when you delete and reload bodies. Wasn't a problem one off, but it came up during training. This fully resets the simulation and seems to have fixed that bug.